mscore/musescore.cpp

```diff
@@ -3560,15 +3560,11 @@ static void loadScores(const QStringList& argv)
             foreach(const QString& name, argv) {
                   if (name.isEmpty())
                         continue;
-                  MasterScore* score = mscore->readScore(name);
-                  if (score) {
-                        mscore->appendScore(score);
+
+                  Score* score = mscore->openScore(name);
+
+                  if (score)
                         scoresOnCommandline = true;
-                        if(!MScore::noGui) {
```

Comment:
Isn't double-clicking a score also affected by this? It is on the command line in that case, but in GUI mode, as far as I can  tell

Reply:
Yep, I didn't mention it in the commit message, but it also affect such a case.

Reply:
But then it should add the score to the list of recently used ones, shouldn't it? Or does it still?

Reply:
Also while at it, you could change that `foreach()` ;-)

Reply:
the openscore method already does all this, so I just cleaned it up here as unnecessary

Reply:
I see, sorry for the noise then